src/operation/decision_model.jl

```diff
@@ -136,41 +135,20 @@ function DecisionModel{M}(
 end
 
 """
-    DecisionModel(::Type{M},
-    template::ProblemTemplate,
-    sys::PSY.System,
-    optimizer::MOI.OptimizerWithAttributes,
-    jump_model::Union{Nothing, JuMP.Model}=nothing;
-    kwargs...) where {M <: DecisionProblem}
-
 This builds the optimization problem of type M with the specific system and template
 
 # Arguments
 
   - `::Type{M} where M<:DecisionProblem`: The abstract operation model type
-  - `template::ProblemTemplate`: The model reference made up of transmission, devices,
-    branches, and services.
+  - `template::ProblemTemplate`: The model reference made up of transmission, devices, branches, and services.
```

Comment:
```suggestion
  - `template::ProblemTemplate`: The model reference comprised of transmission, devices, branches, and services.
```

Reply:
Most people prefer `made up of` over `comprised of`. And this needs an `is`.